.travis.yml

```diff
@@ -25,6 +25,13 @@ script:
   - python -m pytest
   - docker build -t "corobo" .
   - docker images
+  - >
+    docker run -t -i corobo /bin/sh -c "
```

Comment:
Is there a reason why `-t` and `-i` is needed, there's no human interacting with the pseudo-tty and it's running inside a dumb terminal.